### DISCUSSION TITLE
ImportError: There is no such class as speechbrain.decoders.scorer.CTCScorer

### DISCUSSION THREAD

**Discussion by Rel-pcy** [speechbrain/speechbrain#2434] [Q&A] [answered]
when i use speechbrain, it happens "ImportError: There is no such class as speechbrain.decoders.scorer.CTCScorer”.
my speechbrain version is 0.5.9. i don't know whether it is the issue of version

**Comment by mravanelli** [comment, marked as answer]
Please, update to SpeechBrain 1.0
